### ISSUE TITLE
[EPIC] Rope binding system

### ISSUE THREAD

**Issue by toasted323** [toasted323/CoffeeMud#19]
As a player, I want to be able to use ropes to bind, tie, or knot various objects in the game world, allowing for more interactive and realistic gameplay experiences.

TODO summarize existing skills features to "bind" things, see lasso.

**Comment by toasted323** [comment]
@ParreaBob trying to add details here, do you recall which rope related mechanics are in game already?

- lasso
- bind/unbind MOBs

**Comment by ParreaBob** [comment]
Yes quite a few. Bind/unbind (doesnt really req a rope) rope tricks rope trip lasso rope dismount rope disarm off the top of my head.
